dss/api/subscriptions_v2.py

```diff
@@ -28,7 +28,7 @@ def get(uuid: str, replica: str):
     return subscription, requests.codes.ok
 
 
-@security.assert_security(method='read', groups=['dbio', 'public'])
+@security.assert_security(method='group', groups=['dbio', 'public'])
```

Comment:
Should this be extended to other CRUD actions in subscriptions_v2? i would think for get operations we would also want the same group check, but dont need to refer to a UUID to inspect.

Reply:
Yea, that sounds good! Will make those changes

Reply:
Made changes to `dss/api/{subscriptions_v1,subscriptions_v2,collections}.py` to switch endpoints over to `group` method, since all of those endpoints are already doing their own auth checks comparing the token email and owner email

see commit 258a013